dashboard/client/src/App.tsx

```diff
@@ -122,37 +130,61 @@ const App = () => {
-                  path="/log"
-                />
-                <Route
-                  element={<Logs theme={theme as "light" | "dark"} />}
-                  path="/log/:host"
-                />
-                <Route
-                  element={<Logs theme={theme as "light" | "dark"} />}
-                  path="/log/:host/:path"
-                />
-                <Route element={<NodeDetail />} path="/node/:id" />
-                <Route element={<JobDetail />} path="/job/:id" />
-                <Route element={<CMDResult />} path="/cmd/:cmd/:ip/:pid" />
-                <Route element={<Loading />} path="/loading" />
-              </Route>
-            </Routes>
+                  element={<BasicLayout setTheme={setTheme} theme={theme} />}
+                >
+                  <Route element={<Index />} path="/summary" />
+                  <Route element={<Job />} path="/job" />
+                  <Route element={<Node />} path="/node" />
+                  <Route element={<Actors />} path="/actors" />
+                  <Route element={<Events />} path="/events" />
+                  <Route element={<Metrics />} path="/metrics" />
+                  {/* TODO(aguo): Refactor Logs component to use optional query
+                      params since react-router 6 doesn't support optional path params... */}
+                  <Route
+                    element={<Logs theme={theme as "light" | "dark"} />}
+                    path="/log"
+                  />
+                  <Route
+                    element={<Logs theme={theme as "light" | "dark"} />}
+                    path="/log/:host"
+                  />
+                  <Route
+                    element={<Logs theme={theme as "light" | "dark"} />}
+                    path="/log/:host/:path"
+                  />
+                  <Route element={<NodeDetail />} path="/node/:id" />
+                  <Route element={<JobDetail />} path="/job/:id" />
+                  <Route element={<CMDResult />} path="/cmd/:cmd/:ip/:pid" />
+                  <Route element={<Loading />} path="/loading" />
+                </Route>
+                {/* New IA routes below! */}
+                <Route element={<MainNavLayout />} path="/new">
+                  <Route element={<Navigate replace to="overview" />} path="" />
+                  <Route element={<OverviewPage />} path="overview" />
+                  <Route element={<NewIAClusterPage />} path="cluster">
+                    <Route element={<Nodes newIA />} path="" />
+                    <Route element={<NodeDetailPage />} path="nodes/:id" />
+                  </Route>
+                  <Route element={<NewIAJobsPage />} path="jobs">
+                    <Route element={<JobList newIA />} path="" />
+                    <Route element={<JobDetailPage />} path=":id" />
+                  </Route>
+                  <Route element={<NewIALogsPage />} path="logs">
```

Comment:
do these paths need the `/` prefix?

Reply:
No, these are relative routes. Nested below their parent route